nixos/modules/services/web-servers/nginx/gitweb.nix

```
@@ -17,13 +18,45 @@ in
       default = false;
       type = types.bool;
       description = ''
-        If true, enable gitweb in nginx. Access it at http://yourserver/gitweb
+        If true, enable gitweb in nginx.
+      '';
+    };
+
+    location = mkOption {
+      default = "/gitweb";
+      type = types.str;
+      description = ''
+        Location to serve gitweb on.
+      '';
+    };
+
+    user = mkOption {
+      default = "nginx";
+      type = types.str;
+      description = ''
+        User that the CGI process will belong to. (Default almost surely will do.)
```

```suggestion
        Existing user that the CGI process will belong to. (Default almost surely will do.)
```
This requires the user to already exist, so better make that clear.